ruuk/script.module.addon.signals
# Assorted fixes

This PR includes:
- Fix a bug caused by the dot in the return signal name
- Replace RECEIVER global with static variable
- Replace getAddonInfo() call with _addon_id() using static variable
- Fix SPDX to LGPL-2.1-or-later
- ~Switch to using unicode_literals for Python 2~
- Add _to_unicode() function
- Remove unused CallHandler.data
- Use selective imports
    
And also cosmetic/testing improvements:
- Add docstrings to all methods and functions
- Remove disabled import-error and missing-docstring pylint tests
- A codecov.yml file for code coverage
- Add coverage support to Travis config
- Small fixes to setup.py and tox config
- Allow PYTHON to be set for Makefile
- Unit test fix
- Use snake_case for internal functions (no impact)
- Add tests for registerSlot(), sendSignal() and unregisterSlot()
- Add tests for makeCall(), registerCall() and returnCall()

See Travis for test results:
https://travis-ci.org/dagwieers/script.module.addon.signals
And Codecov for coverage results (we have 99% testing coverage!):
https://codecov.io/gh/dagwieers/script.module.addon.signals/branch/assorted-fixes/commits
